news/2018-03-18-featured-artist-track-updates.md

```diff
@@ -0,0 +1,244 @@
+
+### All In One
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/3/previews/385.mp3" type="audio/mpeg">
+</audio>
+
+### Burn Out
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/3/previews/386.mp3" type="audio/mpeg">
+</audio>
+
+### Feel It
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/3/previews/387.mp3" type="audio/mpeg">
+</audio>
+
+### Winter Vacation
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/3/previews/388.mp3" type="audio/mpeg">
+</audio>
+
+## yuki.
+
+**Two** brand new, osu! exclusive tracks arrive with this update from yuki. [Check out his Featured Artist listing for more details and links to the beatmap files for the new tracks!](https://osu.ppy.sh/beatmaps/artists/4)
+
+### daft
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/4/previews/389.mp3" type="audio/mpeg">
+</audio>
+
+### Hello! World (feat. setsunan)
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/4/previews/390.mp3" type="audio/mpeg">
+</audio>
+
+## Helblinde
+
+Though we've come to learn that Helblinde has retired from releasing under this alias for now, we're proud to help keep his legacy alive with the release of his last EP *Omnia Vanitas*, sporting **four** new tracks to test your mapping mettle against.
+
+Do you have what it takes to bend these tracks to your will? [Check out Helblinde's Featured Artist listing for download links and more information!](https://osu.ppy.sh/beatmaps/artists/5)
+
+### Omnia Vanitas
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/5/previews/393.mp3" type="audio/mpeg">
+</audio>
+
+### Elder Hymn
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/5/previews/392.mp3" type="audio/mpeg">
+</audio>
+
+### Unconquered
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/5/previews/394.mp3" type="audio/mpeg">
+</audio>
+
+### Devourer of All
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/5/previews/391.mp3" type="audio/mpeg">
+</audio>
+
+## dark cat
+
+It turns out that we've had a lingering track sitting nice and snug in the Featured Artist archives since the project's inception, just waiting to be released.
+
+Still just as hot as it was when we first got it, there's now **one** new catstep track for dark cat lovers to feast on.
+
+[Download the new track over at dark cat's Featured Artist listing!](https://osu.ppy.sh/beatmaps/artists/6)
+
+### hot chocolate
+
+<audio controls>
+    <source src="https://assets.ppy.sh/artists/6/previews/405.mp3" type="audio/mpeg">
+</audio>
+
+## Loki/Thaehan
+
+Back again under the Thaehan alias with his latest EP, *OMGWTF !*, osu!'s favourite EDM master brings **ten** new tracks to the fray, featuring sounds from some much-loved franchises!
```

Comment:
osu!'s favourite -> community's favourite?

Reply:
i dont see the point in changing this

Reply:
how about just saying `our` - given the personal tone the article has that'd be less distanced